reactor-core/src/test/java/reactor/util/concurrent/SpscArrayQueueTest.java

```diff
@@ -128,7 +128,7 @@ public void shouldNotRemoveElementOnPeek() {
 
 	@Test
 	@Tag("slow")
-	void objectPadding() {
+	public void objectPadding() {
```

Comment:
the `public` modifier shouldn't be needed for JUnit5 tests anymore

Reply:
fixed